bindings/aws/s3/metadata.yaml

```diff
@@ -0,0 +1,125 @@
+# yaml-language-server: $schema=../../../component-metadata-schema.json
+schemaVersion: v1
+type: bindings
+name: aws.s3
+version: v1
+status: stable
+title: "AWS S3 Bindings"
+urls:
+  - title: Reference
+    url: https://docs.dapr.io/reference/components-reference/supported-bindings/s3/
+binding:
+  output: true
+  operations:
+    - name: create
+      description: "Create blob."
+    - name: get
+      description: "Get blob."
+    - name: delete
+      description: "Delete blob."
+    - name: list
+      description: "List blob."
+capabilities: []
+authenticationProfiles:
+  - title: "AccessKey and secretKey"
```

Comment:
As mentioned in the other PR, we should create a shared auth profile for these

Reply:
@ItalyPaleAle Yes, while I agree with adding the shared/common auth profile, I think it should be addressed as a separate PR.
I went ahead and created an issue for it: **[Component Metadata Schema] Extend to Support Shared Auth Profile** #2935 
At the time this feature is added the various components, like AWS, etc... can be modified to take advantage of the feature.

Unless there are any other changes to this current PR, I think we should just complete it.

Reply:
Let’s do #2935 first. I’d rather have things done correctly right away and merge things once they’re ready. Adding a new auth profile shouldn’t be a lot of work: you need to add it here: https://github.com/dapr/components-contrib/blob/master/.build-tools/pkg/metadataschema/builtin-authentication-profiles.go